packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/provider.tsx

```diff
@@ -71,11 +72,17 @@ const provideAction = <T extends AIAction>(
 
 export function setupAIProvider() {
   // a single workspace should have only a single chat session
-  // workspace-id:doc-id -> chat session id
+  // user-id:workspace-id:doc-id -> chat session id
   const chatSessions = new Map<string, Promise<string>>();
 
   async function getChatSessionId(workspaceId: string, docId: string) {
-    const storeKey = `${workspaceId}:${docId}`;
+    const userId = (await AIProvider.userInfo)?.id;
```

Comment:
What does await do here?

Reply:
a bit strange, if is no user, should throw an error immediately.

Reply:
I guess it should have been defined as a function instead. Right now it is just a value.